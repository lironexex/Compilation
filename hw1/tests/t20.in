// comments 1

// this is a legal single-line comment 
// the is a different single-line comment
/* this is a multi-line comment spread over a single line */
/* this is a multi-line comment 
spread 
over 
multiple 
lines 
*/
