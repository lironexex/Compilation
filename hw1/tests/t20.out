1 COMMENT 1
3 COMMENT 1
4 COMMENT 1
5 COMMENT 1
11 COMMENT 6
